docs/connecting-data/connecting-to-apis-and-external-data/clickup-api.md

```diff
@@ -1,7 +1,168 @@
----
-description: How to Connect DataDistillr to the ClickUp API
----
+# Connecting to ClickUp
 
-# ClickUp API
+## Creating a ClickUp Account
+Set up an account with [ClickUp](https://app.clickup.com/signup){target=_blank}.
 
-Details coming soon!
\ No newline at end of file
+???+ cost
+    There are several different types of accounts:
+
+    - Free
+    - Unlimited - $9 per member per month
+    - Business - $19 per member per month
+    - Business Plus - $29 per member per month
+    - Enterprise
+
+[Details and Comparison](https://clickup.com/pricing){target=_blank}
+
+???+ rlimit "Rate Limits"
+
+    The ClickUp API is limited per OAuth and personal token.
+
+    - Free, Unlimited, and Business Plan - 100 requests per minute per token
+    - Business Plus Plan - 1,000 requests per minute per token
+    - Enterprise Plan - 10,000 requests per minute per token
+
+    If you exceed this rate, you will receive a 429 status code.
+
+## How to Connect DataDistillr to ClickUp
+
+To set up a data source connection for ClickUp, you will need to have:
+
+- A [unique name](/connecting-data/connecting-to-apis-and-external-data/clickup-api/#name) for your data source connection to be used in queries.
+- An [API key](/connecting-data/connecting-to-apis-and-external-data/clickup-api/#api-key) generated through your ClickUp account
+
+### Data Source Form
+
+To locate the ClickUp form, follow the steps in [Connecting Your Data to DataDistillr](../../){target=_blank}. When you get to the window to choose the data source type, select API as shown below.
```

Comment:
Allow this link to open in the same tab instead of opening a new tab since it is internal to the ddr docs

Reply:
Sorry, I was so focused on my reverting that I forgot to check about this.